scripts/restore-backup.sh

```diff
@@ -1,25 +1,30 @@
 #!/bin/bash
 set -e
+# Usage: TMP=FOLDER ROOT=FOLDER scripts/restore-backup.sh
+# Example: TMP=/ext/tmp ROOT=/srv/journal-cms scripts/restore-backup.sh
 
+# arguments and environment variables to influence behavior
 TMP="${TMP:-/ext/tmp}"
+ROOT="${ROOT:-/srv/journal-cms}"
+
 # cannot use double quotes to allow bash to resolve * wildcards
 # shellcheck disable=SC2086
 filesArchive=$(find $TMP/*-archive-*.tar.gz)
 # shellcheck disable=SC2086
 databaseArchive=$(find $TMP/*-elife_2_0-mysql.gz)
 
 echo "Extracting $filesArchive"
-tar -xzf "$filesArchive" -C /ext/tmp
+tar -xzf "$filesArchive" -C $TMP
```

Comment:
`$TMP` should be quoted with `"`, in general variables should always be quoted by default